examples/widgets/assets/world.meta

```diff
@@ -0,0 +1,5 @@
+assets/1.0x/world.png
```

Comment:
They're material icons (renamed for brevity). Maybe I could find something better? I really just need a group of images at different resolutions...